app/controllers/news_tematica/news_tematicas_controller.rb

```diff
@@ -103,7 +103,7 @@ def newstematica_klass
     def carga_variables_preview(news_tematica)
       @news_tematica = news_tematica.decorate
       @news_tematica.suscribible_type ||= params[:news_tematica][:suscribible_id].split('-').second
-      todos_los_titulares = @news_tematica.titulares
+      todos_los_titulares = ContenidoEnNewsDecorator.decorate_collection(@news_tematica.titulares)
       @titulares = ContenidoEnNewsDecorator.decorate_collection(todos_los_titulares[0..4])
```

Comment:
I guess that `ContenidoEnNewsDecorator.decorate_collection` can be removed here...